src/readii/feature_extraction.py

```
@@ -276,9 +273,16 @@ def featureExtraction(ctSeriesID):
                                         f"roiImage.GetSize(): {roiImage.GetSize()}"
                                         "Padding segmentation to match."
                                     )
-                                    roiImage = padSegToMatchCT(
-                                        ctDirPath, segFilePath, ctImage, roiImage
-                                    )
+                                    try:
+                                        roiImage = padSegToMatchCT(
```

Ruff

Undefined name `padSegToMatchCT`

(F821)